symplyphysics/laws/electricity/circuits/resistance_in_serial_connection.py

```diff
@@ -42,6 +46,28 @@
         R = \sum_i R_i
 """
 
+# Derive for two resistors.
+# In a parallel connection, the same current flows through the components.
+
+_voltage_expr = solve(_ohm_law.law, _ohm_law.voltage)[0]
+
+_voltage1 = _voltage_expr.subs(_ohm_law.resistance, resistance[1])
+
+_voltage2 = _voltage_expr.subs(_ohm_law.resistance, resistance[2])
+
+_total_voltage = _voltage1 + _voltage2
```

Comment:
Maybe use kirchoff law for this?

Reply:
The Kirchhoff's law is for loops. I'll add a note to make a law for voltage in parallel connected components.